inference-engine/src/mkldnn_plugin/nodes/mkldnn_convert_node.cpp

```diff
@@ -69,9 +69,16 @@ void MKLDNNConvertNode::initSupportedPrimitiveDescriptors() {
 
     config.dynBatchSupport = false;
 
-    // if input and output pointers are not null, then the inp/output tensor descriptors were set using setDescs method, so
+    bool canInitExternalDesc = true;
+    canInitExternalDesc &= (input && output);
+    if (input->getType() & MemoryDescType::Mkldnn)
+        canInitExternalDesc &= input->as<DnnlMemoryDesc>()->isEmpltyExtraData();
+    if (output->getType() & MemoryDescType::Mkldnn)
+        canInitExternalDesc &= output->as<DnnlMemoryDesc>()->isEmpltyExtraData();
+
+    // if input and output pointers are not null and not contain extra data, then the inp/output tensor descriptors were set using setDescs method, so
```

Comment:
First of all we have to check is the descriptors contain the Blocked bit. Then, probably we have to check if they contains extra info. Suggest moving all this stuff under the first if statement `if (input && output)` to avoid extra operations in the default code path.